### PR TITLE
Move message functions to separate package

### PR DESCRIPTION
**Pull request by newhoggy**

### CODE REVIEW

**Comment by AlexeyRaga** [comment]
👍

File: antiope-messages/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# antiope-sqs
```
Comment (dsturnbull) [inline antiope-messages/README.md]: Doesn't match

File: antiope-sqs/src/Antiope/SQS.hs [hunk 2]
```diff
@@ -65,16 +64,15 @@ ackMessages (QueueUrl queueUrl) msgs = do
 -- Extract the "Message" content in the body if have
 messageInBody :: Text -> Maybe Text
 messageInBody body = body ^? key "Message" . _String
+{-# DEPRECATED messageInBody
+    "The export of messageInBody is deprecated.  Use the one from Antiope.Messages instead" #-}
```
Comment (jianwan2016) [inline antiope-sqs/src/Antiope/SQS.hs]: Can we ignore the deprecation and remove the s3 dependency here :)

File: antiope-sqs/package.yaml [hunk 3]
```diff
@@ -33,6 +33,7 @@ dependencies:
 - amazonka-core
 - amazonka-sqs
 - amazonka-s3
+- antiope-messages
```
Comment (dsturnbull) [inline antiope-sqs/package.yaml]: Hmm why does it depend on messages this way?

Comment (dsturnbull) [reply antiope-sqs/package.yaml]: Oh I see later on.

File: antiope-messages/README.md (hunk 1 above)
Comment (newhoggy) [reply antiope-messages/README.md]: I ninja committed a fix.

**Pull request merged**